daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md

```diff
@@ -0,0 +1,139 @@
+---
+type: docs
+title: "Dapr API Logs"
+linkTitle: "API Logs"
+weight: 2000
+description: "Understand how API logging works in Dapr and how to view logs"
+---
+
+This section will assist you in understanding how Dapr API logging works in Dapr, viewing logs.
+
+## Overview
+
+API logs have different, configurable verbosity levels. The API logging is availbale for both HTTP and GRPC calls
+The levels outlined below are the same for both system components and the Dapr sidecar process/container:
+
+1. info
+2. debug
+
+The default level is debug, which provides a balanced amount of information for operating Dapr in normal conditions.
+
+To set the output level, you can use the `--api-log-level` command-line option. For example:
+
+```bash
+./daprd --api-log-level info
+./daprd --api-log-level debug
+```
+
+This will start the Dapr runtime binary with a log level of `info` and `debug` accordingly.
+
+## Logs in stand-alone mode
+
+To set the log level when running your app with the Dapr CLI, pass the `api-log-level` param:
+
+```bash
+dapr run --api-log-level info node myapp.js
+```
+
+As outlined above, every Dapr binary takes a `--api-log-level` argument.
+
+### Viewing Logs on Standalone Mode
+
+When running Dapr with the Dapr CLI, both your app's log output and the runtime's output will be redirected to the same session, for easy debugging.
+For example, this is the output when running Dapr:
+
+The below example is for info level API logging:
+
+```bash
+dapr run --api-log-level info node myapp.js
+ℹ️  Starting Dapr with id Trackgreat-Lancer on port 56730
```

Comment:
This is confusing. Why is the app-id here Trackgreat-Lancer and below it is order-processor.

Reply:
I will change it to order-processor.